app/views/components/_reorderable_list.html.erb

```diff
@@ -0,0 +1,38 @@
+<%
+  items ||= []
+  data_attributes ||= {}
+  data_attributes[:module] = "reorderable-list"
+  input_name ||= "reorderable-list-#{SecureRandom.hex(4)}"
+%>
+
+<%= tag.ol class: "app-c-reorderable-list", data: data_attributes do %>
+  <% items.each do |item| %>
+    <%= tag.li class: "app-c-reorderable-list__item" do %>
+      <%= tag.div class: "app-c-reorderable-list__wrapper" do %>
+        <%= tag.div class: "app-c-reorderable-list__content" do %>
+          <%= tag.p item[:title], class: "app-c-reorderable-list__title" %>
+          <%= tag.p item[:description], class: "app-c-reorderable-list__description" %>
```

Comment:
Most of the app-level components in Content Publisher use helper functions to generate markup. Because we don't have a strict set of rules around how markup should be generated components in govuk_publishing_components are written in a mix of ways. If you pick [the accordion component](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_accordion.html.erb) you'll find the same helpers (aside from one tag, to be fair). I have a note to start discussions around this within the community but it got deprioritised a few times in the past year.

I personally prefer using helpers as they tend to be easier to read and more forgiving when not providing options (by not generating empty attributes, which otherwise would require lots of conditional statements).